### PR TITLE
<FaqView/>

### PR DESCRIPTION
**Pull request by hawkrives**
> Don't merge; this depends on the data changes in #475.

Adds an FAQ view to load the FAQs as hosted on Github Pages.

That way they're as easy to update as merging a PR.

### CODE REVIEW

File: views/faqs/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    backgroundColor: 'transparent',
+  },
+})
+
+export class FaqView extends React.Component {
+  state = {
+    html: faqs,
+  }
+
+  url = "https://stodevx.github.io/AAO-React-Native/faqs.json"
```
Comment (drewvolz) [inline views/faqs/index.js:22]: Will this link work after your data reorg? Current is a 404.

File: docs/faqs.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-  "text": "<h1 id=\"what-s-going-on-with-the-sis-view-\">What&#39;s going on with the SIS view?</h1>\n<p>It&#39;s … struggling. We are currently working on an update that should fix most of the problems.</p>\n<h1 id=\"other-question\">Other Question</h1>\n<p>I&#39;m another question.</p>\n"
+  "text": "<style>\nbody {\n  -webkit-overflow-scrolling: touch;\n  font-family: -apple-system, Roboto, sans-serif;\n  background-color: transparent;\n}\nh1 {\n  font-size: 1.35em;\n}\n</style>\n\n<h1 id=\"what-s-going-on-with-the-sis-view-\">What&#39;s going on with the SIS view?</h1>\n<p>It&#39;s … struggling. We are currently working on an update that should fix most of the problems.</p>\n<h1 id=\"other-question\">Other Question</h1>\n<p>I&#39;m another question.</p>\n"
```
Comment (drewvolz) [inline docs/faqs.json:2]: Do we need to specify webkit, font-family, and background color, and font size in `faqs.json` as well as in `faqs.md`?

File: data/faqs.md [hunk 3]
```diff
@@ -1,3 +1,14 @@
+<style>
+body {
+  -webkit-overflow-scrolling: touch;
```
Comment (drewvolz) [inline data/faqs.md:3]: This is also specified in `faqs.json` ?

**Review by drewvolz — CHANGES_REQUESTED** [review]

File: data/faqs.md (hunk 3 above)
Comment (hawkrives) [reply data/faqs.md:3]: `faqs.json` is the generated version of `faqs.md`. It contains the processed text from faqs.md.

File: views/faqs/index.js (hunk 1 above)
Comment (hawkrives) [reply views/faqs/index.js:22]: Yes. The entire contents of the `docs/` folder on master is published at `https://stodevx.github.io/AAO-React-Native/`.

File: docs/faqs.json (hunk 2 above)
Comment (hawkrives) [reply docs/faqs.json:2]: Halfway. We should never need to manually edit the file in `docs/`; instead, it's generated from the associated file(s) in `data/`.

**Review by drewvolz — APPROVED** [review]

**Review by elijahverdoorn — APPROVED** [review]

**Pull request merged**